### DISCUSSION TITLE
Windows 11 support

### DISCUSSION THREAD

**Discussion by Scuba0070** [SaifAqqad/AHK_MicMute#48] [Q&A] [answered]
I have been searching high and low for something that does what this app does but works with windows 11. I have tried every app I can find and nothing actually mutes the mic. Not even when I click the mic in Windows settings. I am using a USB wireless headset. Does this app work with Windows 11 for anyone?

**Comment by SaifAqqad** [comment, marked as answer]
It does work with windows 11.

Can you explain the issue you have?

**Reply by Scuba0070** [reply]
To put it simply, it does not mute the microphone. Like I said I am using a USB wireless headset. I have made sure that the correct device is selected and even in windows 11 when I go to settings>sound>the device and click start test, the incoming sound when i speak registers on the mic volume slider. i press the hotkey, sound still registers. I think it may be a windows issue (or device issue perhaps) because when I click the mic in that same test it still doesn't mute independent of your app.

**Reply by SaifAqqad** [reply]
That sounds like an issue with your microphone

was it working in windows 10?

**Reply by Scuba0070** [reply]
Never tried it in windows 10. I just used the hard mute switch on the headset but the convenience of a hotkey would be great for gaming.

**Reply by SaifAqqad** [reply]
In that case I think the headset is forcing the state of that mute switch in windows so if the switch is set to mute, then the microphone will stay muted regardless of the setting in windows

**Reply by Scuba0070** [reply]
You were 100% correct. It was the headset. I replaced it with a different brand and it works perfectly. Thank you for the great software.
